src/astbuilder.nit

```diff
@@ -165,6 +166,32 @@ class ASTBuilder
 		return new ANotExpr.make(expr)
 	end
 
+	# Make a new class (AStdClassdef)
+	fun make_class(mclassdef: nullable MClassDef,
```

Comment:
why a AStdClassdef is needed? Can't you just use a MClassDef ?
AST elements are needed to build and inject statements, expression or the body of property definitions. But why the syntactic body of a classdef is needed and by whom ?

Reply:
MClassDef is not enough. I explain myself, in the interpreter and the compiler when we make a `new A` it's equivalent to an implicit call to default_init. But, by default no implementation exists for this method. This is why we delegate the management of this case to the AClassDef. (see `call` of AClassdef in naive_interpreter:1628)

The second argument in favor of creating an AClassDef is that it is easier to debug the AST. Each entity is linked in the AST when we display the AClassDef with `dump tree` we can easily see the set of properties linked to our class.